src/sorting/src/quick_sort.cairo

```diff
@@ -0,0 +1,67 @@
+//! Quick sort algorithm
+use alexandria_data_structures::vec::{Felt252Vec, VecTrait};
```

Comment:
why do you need that ?

Reply:
`Felt252Vec` is imported because it abstracts `dict` to `vec` to allow in place modification of arrays, which is essential for quick sort algorithm.

`VecTrait` is imported to hint the compiler. In practice, if we don't import `VecTrait`, the compiler will fail and complain: error: Method `len` not found on type "alexandria_data_structures::vec::Felt252Vec::<T>". Did you import the correct trait and impl?